src/main/java/org/openstreetmap/atlas/utilities/configuration/StandardConfiguration.java

```diff
@@ -222,4 +256,54 @@ private Object resolve(final String key, final Map<String, Object> currentContex
         }
         return null;
     }
+
+    @SuppressWarnings("unchecked")
+    private Optional<Map<String, Object>> readConfigMapFromJSON(final byte[] readBytes)
```

Comment:
`Config` -> `Configuration` to avoid abbreviations.

Reply:
👍